update.py

```diff
@@ -53,20 +53,36 @@ def import_sheet(update_time, sheet, range, filename, **kwargs):
     write_timestamp_file(filename=filename, old_hash=old_hash)
 
 
-def computeMunicipalities(update_time):
-    filename = 'csv/municipality.csv'
+def computeMunicipalityCases(update_time):
+    filename = 'csv/municipality-cases.csv'
     print("Processing", filename)
     old_hash = sha1sum(filename)
-    dfRegions = pd.read_csv('csv/regions.csv', index_col='date')
-    dfActive = pd.read_csv('csv/active-regions.csv', index_col='date')
-    dfDeceased = pd.read_csv('csv/deceased-regions.csv', index_col='date')
-    dfRegions.columns = [str(col) + '.cases.confirmed.todate' for col in dfRegions.columns]
+    dfConfirmed = pd.read_csv('csv/municipality-confirmed.csv', index_col='date')
+    dfActive = pd.read_csv('csv/municipality-active.csv', index_col='date')
+    dfDeceased = pd.read_csv('csv/municipality-deceased.csv', index_col='date')
+    dfConfirmed.columns = [str(col) + '.cases.confirmed.todate' for col in dfConfirmed.columns]
```

Comment:
no need to change, but for the future, pythonic way of naming would be `df_confirmed`